### DISCUSSION TITLE
per site/domain settings

### DISCUSSION THREAD

**Discussion by jadedgnome** [gildas-lormeau/SingleFile#1518] [Q&A]
I'd like to be able to manually set the filename based on the domain/ site path.

**Comment by gildas-lormeau** [comment]
You can do this with profiles and auto-settings rules in the options page. As an example, I attached a `settings` file which you can import in SingleFile. If you go to a website containing "example" in the URL (e.g. https://example.com or https://example.org), the date will not be included in the filename.

[singlefile-settings.json](https://github.com/user-attachments/files/16419211/singlefile-settings.json)
